assets/tests/components/ladder.test.tsx

```diff
@@ -62,9 +65,116 @@ test("renders a ladder", () => {
         timepoints={timepoints}
         vehicles={vehicles}
         ladderDirection={ladderDirection}
+        selectedVehicleId={undefined}
       />
     )
     .toJSON()
 
   expect(tree).toMatchSnapshot()
 })
+
+test("highlights a selected vehicle", () => {
+  const timepoints = [{ id: "t0" }, { id: "t1" }, { id: "t2" }]
+  const vehicles: Vehicle[] = [
+    {
+      id: "upward",
```

Comment:
I think this test might only need 1 vehicle, which could make the test shorter and easier to read.

Reply:
I'm going to leave 2 to demonstrate others not being selected.